src/content/docs/kubernetes-pixie/k8s-otel/advance-config.mdx

```diff
@@ -0,0 +1,266 @@
+      deployment:
+        envsFrom:
+          - secretRef:
+              name: <Your Secret Name>
+      ```
+  
+
+
+## Add custom configurations in the Helm chart [#custom-config]
+
+The `extraConfig` sections within the `values.yaml` file provide a powerful way to extend the functionality of both the `daemonset` and `deployment` collectors. Depending on your specific requirements, you can choose either collector to apply additional configurations, allowing you to tailor your monitoring experience.
+
+These options offer flexibility for integrating specific settings not included by default. 
+
+{/* For a comprehensive list of receivers, processors, exporters, and pipelines that you can reuse in your configurations, refer to [](#how-works). */}
+
+For more details, always review the `values.yaml` file in the [Helm chart repository](https://github.com/newrelic/helm-charts/blob/master/charts/nr-k8s-otel-collector/values.yaml).
+
+
+## Enable Prometheus service discovery [#enable-prom-sd]
+
+
+To enable Prometheus service discovery within your Kubernetes cluster, use the `extraConfig` sections in your `deployment` collector's configuration. This allows the OpenTelemetry Collector to automatically discover and scrape metrics from services annotated with `prometheus.io/scrape`.
+
+Here's an example configuration snippet to set up service discovery using the standard `prometheus.io/scrape` annotation:
+
+```yaml
+
+  extraConfig:
+    receivers:
+      prometheus/discover:
+        config:
+          scrape_configs:
+            - job_name: "auto-discovered-services"
+              scrape_interval: 30s  # Set the scrape interval to 30 seconds
+              kubernetes_sd_configs:
+                - role: pod
+              relabel_configs:
+                - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
+                  action: keep
+                  regex: true
+                - source_labels: [__meta_kubernetes_pod_label_app]
+                  action: drop
+                  regex: kube-state-metrics
+                - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
+                  action: replace
+                  target_label: __address__
+                  separator: ;
+                  regex: (.+):(?:\d+);(.*)
+                  replacement: $1:$2
+                - action: replace
+                  target_label: job_label
+                  replacement: auto-discovery
+    processors:
+    exporters:
+    connectors:
+    pipelines:
+      metrics/prom_auto_discover:
+        receivers:
+          - prometheus/discover
+        processors:
+          - resource/metrics
+          - k8sattributes/ksm
+          - cumulativetodelta
+          - batch
+        exporters:
+          - otlphttp/newrelic
+
+```
+
+The `prom_auto_discover` pipeline employs several recommended processors to enhance your telemetry data's efficiency and relevance:
+
```

Comment:
This items are relevant not only to the Prometheus service discovery example but to any additional config included